### ISSUE TITLE
Add Ability to display an entry's `notes`

### ISSUE THREAD

**Issue by erichiller** [PSKeePass/PoShKeePass#90]
It would be quite useful to display the `notes` field from an entry. It could be a flag on `-plaintext` or a setting for the database's profile. But beyond `username` and `password` , the `notes` field is absolutely critical (at least for me).

Also, this module is MUCH appreciated. Thanks!

**Comment by jkdba** [comment]
Hi @erichiller, thank you for the feedback and praise it is well received.

The functionality that you mention is already part of the `-AsPlainText` option. It is just not part of the Default Property Set. This mean that is won't be displayed by default when outputting the entries results. You can always access the notes property by piping to the `select-object` function like so.

```powershell
Get-KeePassEntry <options> -AsPlainText | Select-Object -Property Title,UserName,Password,FullPath,Notes
```

To see other properties that are exposed via the `-AsPlainText` option pipe the results to the `Get-Member` Command.

```powershell
Get-KeePassEntry <options> -AsPlainText | Get-Member
```

In this case, it will return a list with the following properties:

> Uuid
> CreationTime
> Expires
> ExpireTime
> LastAccessTime
> LastModificationTime
> LocationChanged
> Tags
> Touched
> UsageCount
> ParentGroup
> FullPath
> Title
> UserName
> Password
> URL
> Notes
> IconId

If you think this should be a default property let me know and I will look into adding it.

**Comment by erichiller** [comment]
To me it would be a default property, but I suppose that is a matter of opinion. It can also be modified by the end user I suppose to. Since there was not any search functionality or a few other things I desired, I wrote a [function on top of the module](https://github.com/erichiller/omega/blob/master/config/ps_functions.ps1#L18) to do so, and it took care of much of what I was after - your example code helped greatly. So I suppose if enough others agreed I would too.
